### DISCUSSION TITLE
Unstaring a repo removes it from all the lists

### DISCUSSION THREAD

**Discussion by antimatter96** [community/community#8663] [Other Feature Feedback, Questions, & Ideas]
Would be nice to mention this behaviour. Just lost 200+ starred repos :/

**Comment by brianlovin** [comment]
@antimatter96 thanks for the note on this — could you clarify how you lost 200 starred repositories? Currently we do require a repo to be starred before it can be added to a list. If you unstar a repo, we will also remove it from any lists it had been added to.

But in this flow, we should not be doing any kind of automatic unstarring. Any clarification here would be helpful, in case there's a different bug that's affecting people's starred repositories.

**Reply by antimatter96** [reply]
Do you guys log the unstar operation ?
Would you be able to provide them ? Should I contact support ?

**Reply by brianlovin** [reply]
Yes, you could contact support for help here.

**Comment by antimatter96** [comment]
@brianlovin 
I went through my stars
    - added repo to the list from the down caret next the starred button
    - unstar

tl;dr: I did not know if this

>  If you unstar a repo, we will also remove it from any lists it had been added to.

 
I guess if you display the number of repos in the lists in bracket next to their name, even that would be helpful inn detecting if something is off

**Reply by jsoref** [reply]
I almost did exactly the same thing. I was trying to convert my ⭐ s into 📝a todo list. And then remove the items from ⭐️ because 99% of the ⭐ d items I had were 📝todo items. They aren't "I ❤️ this", they're "I intend to file PRs against this".

Thankfully after one or two I quickly discovered that unstarring removes items from all lists, and thus I was able to stop & undo w/o losing all of my items.

**Comment by Rock79w** [comment]
Freaks out of issue fix
